.github/workflows/ci.yml

```diff
@@ -12,3 +12,9 @@ jobs:
     with:
       # Turn phpcoverage off because it causes a segfault
       phpcoverage_force_off: true
+      phpunit_skip_suites: framework-cache-only
+      # Ensure we test all in-memory cache factories provided in core
+      extra_jobs: |
+        - phpunit: true
+          phpunit_suite: framework-cache-only
+          install_inmemory_cache_exts: true
```

Comment:
This works with the two CI PRs to ensure we install everything we need to be able to include all three in-memory cache adapters in the new `DefaultCacheFactoryTest` test.